bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -387,9 +383,7 @@ <h2 class="heading">{{ _('Don’t leave home without being in Sync') }}</h2>
         <div id="download-content">
           <h3>{{ _('Choose Firefox') }}</h3>
           <div id="send-to-device-footer">
-            <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-link-type="download" data-download-os="Android" data-alt="{{ _('Get Firefox for Android') }}">
-              {{ high_res_img('firefox/android/btn-google-play.png', {'alt': _('Get it on Google Play'), 'width': '152', 'height': '45', 'l10n': True}) }}
-            </a>
+            {{ google_play_button(class_name='dl-button') }}{# data-alt="{{ _('Get Firefox for Android') }}" #}
```

Comment:
I believe it was for an A/B that may show an alternate string. Seeing as the string exists and is being localized, I think we should probably still retain it here.